stdlib/2.7/typing.pyi

```diff
@@ -174,6 +175,15 @@ class ValuesView(MappingView, Iterable[_VT_co], Generic[_VT_co]):
     def __contains__(self, o: object) -> bool: ...
     def __iter__(self) -> Iterator[_VT_co]: ...
 
+class ContextManager(Generic[_T]):
```

Comment:
Why did you add ContextManager to the 2.7/typing.pyi? It's not in the python2.7/typing.py in the [python/typing](https://github.com/python/typing) repo. And even for Python 3 it's only present from 3.6 on (because it derives from contextlib.AbstractContextManager which is new in 3.6).